src/components/itemBrowser/ItemBrowser.tsx

```diff
@@ -204,6 +204,13 @@ function ItemBrowser<
 
     const getKey = useCallback((str: string, idx: number) => encodeURI(`${str},${idx}`), []);
 
+    const formatRelativePath = useCallback((item: T) => {
```

Comment:
You're not using it in any `deps` array, so useCallback is useless

Reply:
done 4751e1dbcb2c0eb8516b8116398baaf62856fc94